server/server.go

```diff
@@ -338,6 +339,10 @@ func (s *Server) Shutdown(ctx context.Context) error {
 		}
 	}
 
+	if s.config.EventedPLEG {
+		close(s.ContainerEventsChan)
```

Comment:
any reason we can't unconditionally close this?

Reply:
Close of a `nil` channel might `panic`

Reply:
Close of a `nil` channel results in a `panic`
We are [creating](https://github.com/cri-o/cri-o/pull/6404/files#diff-78f42ba40d0f10b08c73b7e6bb8376f398e249c963cf549e89591d6b6826b9a4R443) this channel only in the case of enabled Evented PLEG scenario.

Reply:
can you put a comment about that to make that clearer?